GitVersion.yml

```diff
@@ -2,6 +2,4 @@ assembly-versioning-scheme: Major
 next-version: 6.0
 branches:
   develop:
-    tag: alpha
-  releases?[/-]:
```

Comment:
Is this on purpose? (in core we have RC release branches since we felt that would be the appropriate default)